src/mame/drivers/ksys573.cpp

```diff
@@ -375,6 +376,90 @@ G: gun mania only, drives air soft gun (this game uses real BB bullet)
 
 #define ATAPI_CYCLES_PER_SECTOR ( 5000 )  // plenty of time to allow DMA setup etc.  BIOS requires this be at least 2000, individual games may vary.
 
+// #define JVS_DEBUG
+
+/*
+ * Class declaration for jvs_master
+ */
+
+DECLARE_DEVICE_TYPE(JVS_MASTER, jvs_master)
+
+class jvs_master : public jvs_host
```

Comment:
The names for this device (class name, device type name, short name and description) are too generic for something driver-specific like this.